manywheel/build_common.sh

```diff
@@ -163,10 +163,19 @@ else
 fi
 
 echo "Calling setup.py bdist at $(date)"
-time CMAKE_ARGS=${CMAKE_ARGS[@]} \
-     EXTRA_CAFFE2_CMAKE_FLAGS=${EXTRA_CAFFE2_CMAKE_FLAGS[@]} \
-     BUILD_LIBTORCH_CPU_WITH_DEBUG=$BUILD_DEBUG_INFO \
-     python setup.py bdist_wheel -d /tmp/$WHEELHOUSE_DIR
+if [[ "$DISABLE_RCCL" = 1 ]]; then
+    echo "Disabling NCCL/RCCL in pyTorch"
+    time CMAKE_ARGS=${CMAKE_ARGS[@]} \
+        EXTRA_CAFFE2_CMAKE_FLAGS=${EXTRA_CAFFE2_CMAKE_FLAGS[@]} \
+        BUILD_LIBTORCH_CPU_WITH_DEBUG=$BUILD_DEBUG_INFO \
+        USE_NCCL=0 USE_RCCL=0 USE_KINETO=0 \
+        python setup.py bdist_wheel -d /tmp/$WHEELHOUSE_DIR
+else
+    time CMAKE_ARGS=${CMAKE_ARGS[@]} \
+        EXTRA_CAFFE2_CMAKE_FLAGS=${EXTRA_CAFFE2_CMAKE_FLAGS[@]} \
+        BUILD_LIBTORCH_CPU_WITH_DEBUG=$BUILD_DEBUG_INFO \
+        python setup.py bdist_wheel -d /tmp/$WHEELHOUSE_DIR
+fi
```

Comment:
```suggestion
if [[ "$RCCL_ENABLED" = 0 ]]; then
    echo "Disabling NCCL/RCCL in pyTorch"
fi
time CMAKE_ARGS=${CMAKE_ARGS[@]} \
    EXTRA_CAFFE2_CMAKE_FLAGS=${EXTRA_CAFFE2_CMAKE_FLAGS[@]} \
    BUILD_LIBTORCH_CPU_WITH_DEBUG=$BUILD_DEBUG_INFO \
    USE_NCCL=${RCCL_ENABLED:-1} USE_RCCL=${RCCL_ENABLED:-1} USE_KINETO=${RCCL_ENABLED:-1} \
    python setup.py bdist_wheel -d /tmp/$WHEELHOUSE_DIR
```

Reply:
@pruthvistony Can we change the polarity of the variable we specify in the CI job, so we can have this simpler diff wrt upstream? I would like to minimize lines of change.

Reply:
Sorry for the rework, I was discussing with Jeff offline what the best course of action would be :)

Reply:
Since NCCL/RCCL is by default always enabled, in my opinion reversing the polarity of the variable causes more code readability issue. Trade off is few more lines of code over readability. Prefer to keep the code as per Jeff comments.

Reply:
Okay, thanks for considering